src/JsonSchema/Constraints/BaseConstraint.php

```diff
@@ -53,7 +53,7 @@ public function addError(ConstraintError $constraint, JsonPointer $path = null,
             'pointer' => ltrim(strval($path ?: new JsonPointer('')), '#'),
             'message' => ucfirst(vsprintf($message, array_map(function ($val) {
                 if (is_scalar($val)) {
-                    return $val;
+                    return var_export($val, true);
```

Comment:
Extra credit: I noticed that the const error string was producing this message for bool type:

> Does not have a value equal to 1

instead of 

> Does not have a value equal to true